packages/common-all/src/api.ts

```diff
@@ -257,13 +266,34 @@ abstract class API {
     }
     return payload;
   }
+
+  async _makeRequestRaw(args: IDoRequestArgs) {
```

Comment:
Why is this method necessary? Is this not roughly the same logic as `_makeRequest()` without `payloadData` specified?

Reply:
the return signature is different. payload is an object of {error: Error, data: T}. when returning a raw request, we are returning either Error|Buffer